src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -304,6 +304,34 @@ class GoogleExtensionSpec(_system: ActorSystem) extends TestKit(_system) with Fl
 
   }
 
+  it should "include username, subject ID, and apps domain in proxy group email" in {
+    val appsDomain = "test.cloudfire.org"
+    val subjectId = "0123456789"
+    val username = "foo"
+
+    val config = googleServicesConfig.copy(appsDomain = appsDomain)
+    val googleExtensions = new GoogleExtensions(null, null, null, null, null, null, null, config, null, null)
+
+    val user = WorkbenchUser(WorkbenchUserId(subjectId), WorkbenchEmail(s"$username@test.org"))
+
+    val proxyEmail = googleExtensions.toProxyFromUser(user).value
+    proxyEmail shouldBe "foo_0123456789@test.cloudfire.org"
+    proxyEmail should include (username)
+    proxyEmail should include (subjectId)
+    proxyEmail should include (appsDomain)
```

Comment:
seems redundant to check "proxyEmail shouldBe ..." and all the should includes

Reply:
It technically is. However the requirement is (and the description of the test case indicates) that each of those items is in the email address. Having the elements as separate strings along with the individual asserts makes it clear that those facts are true, more so than having to mentally match the overall `shouldBe`.

An alternative would be simply:
```
proxyEmail shouldBe s"${username}_$subjectId@$appsDomain"
```
which feels a little too much like a re-implementation of the function but is probably okay.